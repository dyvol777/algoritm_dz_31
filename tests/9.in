5 8
1 1 0 0 0 3 a
0 0 1 1 0 3 b
0 0 1 0 1 5 c
0 1 0 0 1 5 d
1 1 1 0 0 4 e
1 0 0 1 1 5 f
1 1 0 1 1 5 j
0 1 1 1 1 5 h