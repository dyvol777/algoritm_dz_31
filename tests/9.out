something going wrong! - bad input!
